ui/src/stylesheets/toc.scss

```diff
@@ -43,6 +43,19 @@
   padding-right: var(--sm);
   overflow: hidden;
   text-overflow: ellipsis;
+  font-family: var(--sans-serif);
+  font-weight: var(--weight-bold);
+  font-size: var(--md);
+  opacity: .8;
+
+  &:hover {
```

Comment:
This hover is applying to all the subitems of an item as well, because they are all inside the `li` element.